tests/fixtures/optimizer/eliminate_join_marks.sql

```diff
@@ -0,0 +1,41 @@
+ 
+# title: Constant in join  
+# dialect: oracle
+SELECT T1.d, T2.c FROM T1, T2 WHERE T1.x = T2.x (+) and T2.y (+) > 5;
+SELECT T1.d, T2.c FROM T1 LEFT JOIN T2 ON T1.x = T2.x AND T2.y > 5;
+  
+# title: Null in Join Predicte  
```

Comment:
There's a typo here, "Predicte" needs to be predicate

Reply:
file removed